frontend/e-buddy/src/components/chatbot/ChatTrigger.tsx

```
@@ -0,0 +1,23 @@
+
+export const ChatTrigger = ({ customization, onClick }: ChatTriggerProps) => {
+  return (
+    <button
+      className="fixed bottom-5 shadow-lg rounded-full p-4 flex items-center justify-center z-50 transition-all hover:scale-105"
+      style={{ 
+        backgroundColor: customization.primaryColor || '#9b87f5',
+        right: customization.position === 'right' ? '20px' : 'auto',
+        left: customization.position === 'left' ? '20px' : 'auto'
+      }}
+      onClick={onClick}
+    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)